### PR TITLE
[DX-1633] Improve for better search results

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>
To improve the Algolia's search results
---

### Preview Link

[preview](https://deploy-preview-5294--tyk-docs.netlify.app/docs/nightly/tyk-self-managed/install)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-self-managed/install.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 date: 2017-03-15T15:01:42Z
-title: "Installation options"
+title: "Installation Options for Tyk Self-Managed"
 tags: ["Tyk Stack", "Self-Managed", "Installation", "Installation options", "Containerized installation", "Open system installation"]
-description: "How to install the Self-Managed Tyk Stack"
+description: "Installation options for the Self-Managed Tyk Stack"
```
Comment (letzya) [inline tyk-docs/content/tyk-self-managed/install.md]:
```suggestion
description: "Landing page for all the installation options we provide for the Self-Managed Tyk Stack"
```

**Review by dcs3spp — APPROVED** [review]
PR LGTM thanks Yaara

**Review by yurisasuke — APPROVED** [review]

**Pull request merged**